iceoryx_posh/test/integrationtests/test_publisher_subscriber_communication.cpp

```diff
@@ -126,6 +157,111 @@ TEST_F(PublisherSubscriberCommunication_test, AllSubscriberInterfacesCanBeSubscr
     }
 }
 
+TEST_F(PublisherSubscriberCommunication_test, SubscriberRequiringHistoryDoesNotConnectToPublisherWithoutHistorySupport)
+{
+    ::testing::Test::RecordProperty("TEST_ID", "31cbd36d-32f1-4bc7-9980-0cdf5f248035");
+
+    constexpr uint64_t historyRequest = 1;
+    constexpr uint64_t historyCapacity = 0;
+    constexpr bool requireHistory = true;
```

Comment:
How about calling this `requireHistorySupport`? It would make the intention more obvious.

Reply:
Sure. General rule should be: the larger the scope, the more expressive a name needs to be. Interfaces need expressive names as well, local scope not so much if it means the following lines become more readable.